docs/source/api.rst

```diff
@@ -1,2 +1,70 @@
-Using the API
-=============
+Using the GameBench API Client
+==============================
+
+Installation
+------------
+
+The GameBench API Client can be install using pip:
+
+.. code-block:: python
+
+    pip install gamebench-api
+
+It can also be found on the `GitHub`_ page.
+
+
+Setup
+-----
+
+Add the username and password for the GameBench account you are using to the 'global_settings.py'
+module.
+
+.. code-block:: python
+   :linenos:
+
+    GAMEBENCH_CONFIG = {
+        'url': 'https://api.production.gamebench.net',
+        'api_version': '/v1',
+        'username': 'john.smith@example.com',
+        'password': 'password',
+        'company_id': '',
+    }
+
+See :doc:`global_settings` for for information on this module.
+
+
+Importing
```

Comment:
What do you think about renaming this section to, "Basic Usage" and providing an example of what is returned?

Reply:
Got the name changed, and will get an example of what is returned.

Reply:
Got an example of what the DataFrame looks like when it is returned.  Check it out at your leisure and let me know what you think!